src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDaoMockTest.java

```diff
@@ -46,48 +43,46 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
 
-public class DynamoSchedulePlanDaoMockTest {
+public class DynamoSchedulePlanDaoMockTest extends Mockito {
```

Comment:
We need the following tests

* createSchedulePlan sets the studyKey
* updateSchedulePlan sets the studyKey
* general tests for createSchedulePlan
* general tests for updateSchedulePlan
* general tests for getSchedulePlan

Reply:
Added